### DISCUSSION TITLE
Shortcuts

### DISCUSSION THREAD

**Discussion by gusmanb** [gusmanb/ZXBasicStudio#10] [Ideas]
Post any action that you think it should be possible to access using a shortcut.

-----------------------
Cut
Copy
Paste
Comment
Uncomment
Fold all
Unfold all

**Comment by Duefectu** [comment]
Move line up/dow
Run
Debug
Format
Undo
Redo

**Reply by githubfito** [reply]
Emulator FullScreen (alt+enter=default) :-)
